### PR TITLE
replace `run_cmd` with `run_shell_cmd` in custom easyblock for NAMD (`namd.py`)

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyblocks/n/namd.py [hunk 1]
```diff
@@ -208,14 +208,14 @@ def test_step(self):
                 'testdir': os.path.join(self.cfg['start_dir'], self.namd_arch, 'src', 'alanin'),
                 'testopts': self.cfg['testopts'],
             }
-            out, ec = run_cmd(cmd, simple=False)
-            if ec == 0:
+            res = run_shell_cmd(cmd)
```
Comment (bartoldeman) [inline easybuild/easyblocks/n/namd.py:211]: isn't the exit code always 0 here since it would raise an exception otherwise?

**Review by bartoldeman — DISMISSED** [review]

File: easybuild/easyblocks/n/namd.py [hunk 2]
```diff
@@ -208,14 +208,14 @@ def test_step(self):
...
+            if res.exit_code == 0:
```
Comment (boegel) [inline easybuild/easyblocks/n/namd.py:212]: Useless `if`, let's get rid of it

File: easybuild/easyblocks/n/namd.py (hunk 1 above)
Comment (branfosj) [reply easybuild/easyblocks/n/namd.py:211]: 5a4e5f7

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _NAMD-2.14-foss-2023a-mpi.eb_ 

Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3555.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.6.8
See https://gist.github.com/boegel/f536ccf823166d66830a7db254693132 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**